### PR TITLE
Rename OnlineWallet to Wallet

### LINKED ISSUE #81: Remove "OfflineWallet" and rename "OnlineWallet" to Wallet

### PR DESCRIPTION
**Pull request by artfuldev**
and remove OfflineWallet

Fix #81

### CODE REVIEW

**Comment by artfuldev** [comment]
@notmandatory This requires downstream changes for anyone using the new API

File: src/lib.rs [hunk 1]
```diff
@@ -250,18 +230,18 @@ impl OnlineWallet {
         };
         let database = AnyDatabase::from_config(&any_database_config)?;
         let blockchain = AnyBlockchain::from_config(&any_blockchain_config)?;
-        let wallet = Mutex::new(Wallet::new(
+        let _wallet = Mutex::new(BdkWallet::new(
```
Comment (notmandatory) [inline src/lib.rs:233]: usually the _ is used for variables that aren't used, so I think this should be renamed back to `wallet` or possibly `wallet_mutex` if you want to be more descriptive.

**Comment by artfuldev** [comment]
@notmandatory would you like to merge this?

**Review by notmandatory — APPROVED** [review]
utACK d09c8a41604606ff5c40eb37779f7204c849e75a

I'll merge this one now

**Pull request merged**